constants/events.ts

```diff
@@ -1,62 +1,66 @@
 import { Event } from "@/types/types";
 
 import event from "@/public/placeholder/event.png";
+import dataViz from "@/public/events/dataViz.png";
+import bot from "@/public/events/bot.png";
+import leetcoding from "@/public/events/leetcoding.png";
 
 export const UPCOMING_EVENTS: Event[] = [
```

Comment:
- [ ] Make sure each description is just a couple lines of text in paragraph form. Otherwise it looks weird in each card.
- [ ] Truncate the description text if it goes over 3 lines

Reply:
I also uncommented the events section on the homepage so you can see what it looks like